pkg/util/merr/errors.go

```diff
@@ -105,6 +105,12 @@ var (
 	ErrMqTopicNotEmpty = newMilvusError("topic not empty", 1301, false)
 	ErrMqInternal      = newMilvusError("message queue internal error", 1302, false)
 
+	// Privilege related
+	// this operation is denied because the user not authorized, user need to login in first
+	ErrPrivilegeNotAuthenticated = newMilvusError("privilege not authenticated", 1400, false)
```

Comment:
NIT: "privilege not authenticated" is way too weird, maybe "not authenticated" is no enough